.github/workflows/ci.yaml

```diff
@@ -141,8 +139,38 @@ jobs:
           if (( $(echo "$SKIPPED > $MAX_SKIPPED" | bc -l) )); then echo "Fail: number of skipped statements '$SKIPPED' is higher than configured '$MAX_SKIPPED'" >&2; exit 1; fi
           echo "Number of skipped statements '$SKIPPED' is within configured '$MAX_SKIPPED'"
 
+  report-coverage:
+    needs: coverage
+    runs-on: ubuntu-latest
+    env:
+      COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+    name: "report: coverage"
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3.1.0
+
+      - name: Setup node
+        uses: actions/setup-node@v3.5.1
+        with:
+          node-version: 16.14.x
+          cache: "npm"
+
+      - name: npm install
+        run: npm install
+
+      - name: Download code coverage results
+        uses: actions/download-artifact@v2
+        with:
```

Comment:
![image](https://github.com/bosonprotocol/boson-protocol-contracts/assets/64400885/c84fb0cd-f5bc-428d-8ec0-81c7adb00609)

Can you set node version to 16 to get rd of this warning?

Reply:
done